tests/test_scalar_coverage.py

```
@@ -0,0 +1,1166 @@
+        a = Scalar([-1., 0., 1.])
+        b = a.sign(zeros=False)
+        self.assertEqual(b[1], 1)  # Zero should become 1
+
+        # Test builtins (line 760-764)
+        a = Scalar(1.)
+        Qube.prefer_builtins(True)
+        b = a.sign()
+        # sign() returns the sign, which for float 1.0 is 1.0 (float), not int
+        # But if it's an integer Scalar, it might return int
+        a_int = Scalar(1)  # Integer
+        b_int = a_int.sign()
```

Ruff (0.14.7)

Local variable `b_int` is assigned to but never used

Remove assignment to unused variable `b_int`

(F841)